website/index.pug

```diff
@@ -1,37 +1,65 @@
 extends partials/site.pug
 
-block vars
-  - var navLinks = []
-  - navLinks.push({ href: 'https://github.com/cginternals/webgl-operate', text: 'GitHub' })
-  - navLinks.push({ href: 'https://webgl-operate.org/doc', text: 'Documentation' })
-
 
 block head
 
-  // link(rel = 'stylesheet' href = 'css/template.css')
-  // script(src = 'js/tempalte.js')
-
 
 block header
 
-  section.container
-   header.jumbatron
-    img(src = 'img/webgl-operate-logo.svg' alt = 'webgl-operate logo').mb-2.w-50.d-none.d-md-block
-    img(src = 'img/webgl-operate-logo.svg' alt = 'webgl-operate logo').mb-2.d-block.d-md-none
-    p.lead webgl-operate (or gloperate for WebGL) is a TypeScript based WebGL rendering framework.
+  header.container.mb-4
+    .row
+      .col-lg-6.col-12.pb-5
+        img(src = 'img/webgl-operate-logo.svg' alt = 'webgl-operate logo').mb-2.w-100.d-none.d-md-block
+        img(src = 'img/webgl-operate-logo.svg' alt = 'webgl-operate logo').mb-2.d-block.d-md-none
+
+        p.lead Build high-performance rendering applications with this use case agnostic, TypeScript based WebGL engine.
+        p.lead.mb-4 webgl-operate is an open source engine that enables hardware-accelerated 2D and 3D rendering and computation. It is highly extensible and facilitates arbitrary rendering approaches such as forward rendering, offscreen computing, multi-frame sampling, raymarching raytracing, or progressive pathtracing.
+
+        a(href = '/doc/' role = 'button').btn.btn-lg.btn-primary.ml-4.mr-2 Get Started
+        a(href = 'https://github.com/cginternals/webgl-operate/archive/v0.4.2-beta.zip' role = 'button').btn.btn-lg.btn-primary.ml-2 Download
```

Comment:
`v0.4.2-beta.zip`?

Reply:
hmm.. some npm packaging mess, i tried to use postfixes/tags... should be resolved with next release